server/lib/routes/get-index.js

```diff
@@ -2,6 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var crypto = require('crypto');
+
+var activityEventQueryParams = [
+  'campaign',
```

Comment:
campaign is not used anymore, we need to get rid of it everywhere, there is an issue open for the front-end